docs/en_US/AdvancedFeature/MultiPhase.md

```diff
@@ -22,7 +22,10 @@ It is pretty simple to use multi-phase in trial code, an example is shown below:
     # ...
     for i in range(5):
         # get parameter from tuner
+        # nni.get_next_parameter returns None if there is no more hyper parameters can be generated by tuner.
```

Comment:
Could we add to a more conspicuous position so that the user could see it?

Reply:
Thanks, updated.